src/org/parosproxy/paros/control/MenuFileControl.java

```diff
@@ -461,22 +462,6 @@ public String getDescription() {
                 log.error(e.getMessage(), e);
     		}
 	    }
-	    */
-		String fileName = session.getFileName();
-		
-		if (fileName.endsWith(".session")) {
-		    fileName = fileName.substring(0, fileName.length() - 8);
-		}
-		fileName += "-" + dateFormat.format(new Date()) + ".session";
-		
-		try {
-    	    waitMessageDialog = view.getWaitMessageDialog(Constant.messages.getString("menu.file.savingSnapshot"));	// ZAP: i18n

```

Comment:
The current message is using "menu.file.savingSession" instead of "menu.file.savingSnapshot", should use the latter? (if not the message should be removed, does not seem to be used in other place)